lib/Doctrine/ORM/Configuration.php

```diff
@@ -182,14 +183,21 @@ public function newDefaultAnnotationDriver($paths = [], $useSimpleAnnotationRead
         }
 
         if ($useSimpleAnnotationReader) {
```

Comment:
To be deprecated later on I suppose.

Reply:
The whole method is deprecated already and its replacement does not have that parameter anymore.